__tests__/lint/enforcements-casing-camel-case-error/.redocly.yaml

```diff
@@ -0,0 +1,11 @@
+apiDefinitions:
+  main: ./openapi.yaml
+
+lint:
+  rules:
+    enforcements:
+      - on:
+          - Operation.operationId
+        description: Operation id sould be camelCase
+        casing: PascalCase
```

Comment:
from the file name looks like you're testing camel-case but here you specify PascalCase